applications/ipc_radio/README.rst

```diff
@@ -89,24 +90,20 @@ The following files are available:
 * :file:`overlay-bt_hci_ipc.conf` - Configuration file enabling Bluetooth Low Energy over HCI.
 * :file:`overlay-bt_rpc.conf` - Configuration file enabling Bluetooth Low Energy over RPC.
 
-Building and running
-********************
+.. note::
+   When using Sysbuild, the configuration files are added automatically depending on the Sysbuild configuration.
```

Comment:
```suggestion
   When using sysbuild, the configuration files are added automatically depending on the sysbuild configuration.
```
Global edit for the page: `sysbuild`. Zephyr docs love decapitalized proper names (`west`, not `West`), so we need to follow suit: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/build/sysbuild/index.html -- only the title is capitalized